Description: Update sgmlspl.pl
 Update sgmlspl.pl to reflect change of location of spec-files
Author: Ardo van Rangelrooij <ardo@debian.org>
Bug-Debian: https://bugs.debian.org/58443
Forwarded: not-needed
Last-Update: 2014-04-10
diff --git a/sgmlspl.pl b/sgmlspl.pl
index c91a6cb..ce79562 100755
--- a/sgmlspl.pl
+++ b/sgmlspl.pl
@@ -238,7 +238,7 @@ sub main::sgml {
 package main;
 
 $ARGV = shift;
-unless ($ARGV eq '' || do $ARGV) {
+unless ($ARGV eq '' || do 'sgmlspl-specs/' . $ARGV || do $ARGV) {
     if (!-e $ARGV) {
 	die "FATAL: $ARGV does not exist.\n";
     } elsif (!-r $ARGV) {
